### PR TITLE
Cleanup docstrings for `gammapy.scripts`, `gammapy.stats` and `gammapy.visualization`

### PR DESCRIPTION
**Pull request by cgalelli**
This addresses part of issue #4811 , specifically for `gammapy.scripts`, `gammapy.stats` and `gammapy.visualization`.

### CODE REVIEW

File: gammapy/scripts/info.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def print_info(info, title):
 
 
 def get_info_system():
-    """Get info about user system"""
+    """Get info about user system."""
```
Comment (Astro-Kirsty) [inline gammapy/scripts/info.py:78]:
```suggestion
    """Get information about user system."""
```

File: gammapy/stats/counts_statistic.py [hunk 2]
```diff
@@ -83,12 +83,12 @@ def _repr_html_(self):
...
 
     def info_dict(self):
-        """A dictionary of the relevant quantities
+        """A dictionary of the relevant quantities.
 
         Returns
         -------
         info_dict : dict
-            Dictionary with summary info
+            Dictionary with summary info.
```
Comment (Astro-Kirsty) [inline gammapy/stats/counts_statistic.py:91]:
```suggestion
            Dictionary with summary information.
```

**Review by Astro-Kirsty — COMMENTED** [review]
Thanks @cgalelli !

File: gammapy/visualization/heatmap.py [hunk 3]
```diff
@@ -19,21 +19,20 @@ def plot_heatmap(
...
-    **kwargs
-        All other arguments are forwarded to `imshow`.
+    data : `~numpy.ndarray`
+        Data array.
+    row_labels : list or `~numpy.ndarray`
+        List or array of labels for the rows.
+    col_labels : list or `~numpy.ndarray`
+        List or array of labels for the columns.
+    ax : `matplotlib.axes.Axes`, optional
+         Ax instance to which the heatmap is plotted. Default is None. If None, the current one is used.
```
Comment (registerrier) [inline gammapy/visualization/heatmap.py:29]:
```suggestion
        List or array of labels for the columns.
    ax : `matplotlib.axes.Axes`, optional
        Axis instance to which the heatmap is plotted. Default is None. If None, the current one is used.
```

File: gammapy/visualization/heatmap.py [hunk 4]
```diff
@@ -19,21 +19,20 @@ def plot_heatmap(
...
+    cbar_kw : dict, optional
+        A dictionary with arguments to `matplotlib.Figure.colorbar`. Default is None.
+    cbarlabel : str, optional
+        The label for the color bar. Default is "".
+    **kwargs : dict, optional
+         Other keyword arguments forwarded to `matplotlib.axes.Axes.imshow`.
```
Comment (registerrier) [inline gammapy/visualization/heatmap.py:35]:
```suggestion
        The label for the color bar. Default is "".
    **kwargs : dict, optional
        Other keyword arguments forwarded to `matplotlib.axes.Axes.imshow`.
```

File: gammapy/visualization/heatmap.py [hunk 5]
```diff
@@ -88,21 +87,21 @@ def annotate_heatmap(
...
-    threshold
+    data : `~numpy.ndarray`, optional
+        Data used to annotate. Default is None. If None, the image's data is used.
+    valfmt : str format or `matplotlib.ticker.Formatter`, optional
+        The format of the annotations inside the heatmap. This should either
+        use the string format method, e.g. "$ {x:.2f}"
+        or be a `matplotlib.ticker.Formatter` instance. Default is "{x:.2f}".
+    textcolors : list or `~numpy.ndarray`, optional
+        Two color specifications.  The first is used for
+        values below a threshold, the second for those above. Default is ("black", "white").
```
Comment (registerrier) [inline gammapy/visualization/heatmap.py:98]: Default is given as tuple, whereas argument is a list

**Review by registerrier — APPROVED** [review]
Thanks @cgalelli . See minor comments inline.

**Comment by registerrier** [comment]
Thanks @cgalelli . Merging now.

**Pull request merged**